deployment/GI03FLMB_Deploy.csv

```diff
@@ -1,118 +1,118 @@
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOG065,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGINS-CTDMOG-13432,59.7178,-39.3525,,180,2817,
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOG066,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGINS-CTDMOG-11639,59.7178,-39.3525,,250,2817,
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOG067,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGINS-CTDMOG-12378,59.7178,-39.3525,,350,2817,
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOG068,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOG-11647,59.7178,-39.3525,,500,2817,
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOH069,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-13446,59.7178,-39.3525,,750,2817,
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOH070,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-11598,59.7178,-39.3525,,1000,2817,
+AR21,,AR30-03,,GI03FLMB-RIM01-02-CTDMOH071,4,1,2017-08-07T15:29:00,2018-06-11T19:42:00,CGMGI-03FLMB-00004,,CGINS-CTDMOH-12224,59.7178,-39.3525,,1500,2817,
+AR21,,AR30-03,,GI03FLMB-RIS01-00-SIOENG000,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGCON-SIOCON-12372,59.7178,-39.3525,,30,2817,
+AR21,,AR30-03,,GI03FLMB-RIS01-03-DOSTAD000,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGINS-DOSTAD-00456,59.7178,-39.3525,,30,2817,
+AR21,,AR30-03,,GI03FLMB-RIS01-04-PHSENF000,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGINS-PHSENF-P0102,59.7178,-39.3525,,30,2817,
+AR21,,AR30-03,,GI03FLMB-RIS01-05-FLORTD000,4,1,2017-08-07T15:29:00,2018-06-11T19:36:00,CGMGI-03FLMB-00004,,CGINS-FLORTD-01318,59.7178,-39.3525,,30,2817,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH000,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-10266,59.71427,-39.31329,,1830,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH100,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-11600,59.71427,-39.31329,,2730,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH400,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-11599,59.71427,-39.31329,,2430,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-CTDMOH700,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-10267,59.71427,-39.31329,,2130,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB000,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-11669,59.71427,-39.31329,,1830,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB100,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-12195,59.71427,-39.31329,,2730,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB400,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-11967,59.71427,-39.31329,,2430,2829,
+AR30-03,,AR35-05,,GI03FLMB-RI000-00-VELPTB700,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-VELPTB-11964,59.71427,-39.31329,,2130,2829,

```

Comment:
The OSNAP depths are incorrect (as is the spreadsheet - my bad) they should:
1830 –> 1829
2730 –> 2729
2430 –> 2429
2130 –> 2129

Reply:
corrected